static/src/javascripts/projects/commercial/modules/header-bidding/slot-config.ts

```diff
@@ -47,55 +48,58 @@ const getSlots = (contentType: string): HeaderBiddingSlot[] => {
 		{
 			key: 'right',
 			sizes: hasShowcase
-				? [[300, 250]]
+				? [[adSizes.mpu.width, adSizes.mpu.height]]
```

Comment:
I'm just wondering if we can treat `adSizes.whatever` as an array so:

```suggestion
				? [adSizes.mpu]
```

@Jakeii did some magic around this i.e. `AdSize extends Array` this so might be worth trying out.

Reply:
Thanks Ravi! I got a type error when I tried to do this originally so I'll have a chat with Jake and see what we come up with

Reply:
Nice update! My approval still stands so feel free to merge :)